browser_patches/firefox/juggler/TargetRegistry.js

```diff
@@ -198,7 +198,7 @@ class TargetRegistry {
         setViewportSizeForBrowser(browserContext.defaultViewportSize, tab.linkedBrowser, window);
     };
 
-    const onTabCloseListener = event => {
+    const onTabCloseListener = async event => {
```

Comment:
not sure this is needed!

Reply:
Oops, missed this comment, will fix in a follow-up.